setup.py

```diff
@@ -52,6 +53,14 @@ def read_file(filename, encoding='utf8'):
     'zope.interface',
 ]
 
+# Add pywin32 on Windows platforms to handle low-level system calls.
+# This dependency needs to be added using environment markers to avoid its installation on Linux.
+# However environment markers are supported only with setuptools >= 36.2.
+# So this dependency is not added for old Linux distributions with old setuptools,
+# in order to allow these systems to build certbot from sources.
+if StrictVersion(setuptools_version) >= StrictVersion('36.2'):
+    install_requires.append('pywin32 ; sys.platform == \'win32\'')
+
```

Comment:
I think it might be a good idea to add an `elif` branch here that checks if `bdist_wheel` is in `sys.argv` and if it is, errors out. This would allow us to catch if we're trying to build a wheel with an old version of `setuptools` and prevent us from releasing a bad wheel.

I'm reasonably confident our packagers don't use wheels at all and if they do, it's an easy thing for them to patch.

Reply:
Done.